app/main.py

```diff
@@ -1 +1,35 @@
-# write your code here
+class CommandNameError(Exception):
+    """CommandNameError"""
+
+
+class DoubleNameError(Exception):
+    """DoubleNameError"""
+
+
+class LanCommandError(Exception):
+    """LaneCommandError"""
+
+
+def copy_file(command: str) -> None:
+    split_command = command.split()
```

Comment:
The variable name 'split_command' is not very descriptive. Consider renaming it to something more meaningful, such as 'command_parts' or 'command_elements'.